transforms/helpers/parse-helper.js

```diff
@@ -473,8 +474,44 @@ function getClassName(
   type = ""
 ) {
   const varDeclaration = getClosetVariableDeclaration(j, eoCallExpression);
+  const filePathSplit = filePath.startsWith("app/")
```

Comment:
This should be the case for everyone, even if their pods are in a different directory. The cases this would need to change for is engines, dummy app, and MU.